.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Shell scripts
+
+on:
+  pull_request:
+    paths:
+      - "**/*.sh"
+
+jobs:
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@v0.1.0
```

Comment:
Is it possible to control the version of shfmt as it was in Travis CI?

Reply:
This was just a pre-packaged Action I found that contained both. I could just swap back to individual containers if we want control

Reply:
I vote for more precise control, to make sure the build won't be blocked by upstream changes, thanks.

Reply:
OK, switched back to the individual calls and updated it to the 3.0 version of your container

Reply:
PS: junk PR/branch I used for testing if you want to see it working https://github.com/nschonni/docker-node/pull/2